### PR TITLE
Add installation steps of founationDB in examples/stable

### PR DESCRIPTION
**Pull request by viveksinghggits**
## Change Overview
This PR adds the steps of how we can install foundationDB on our Kubernetes cluster. There are some steps still to be added that explain how we can created backup and then restore that backup using Kanister custom resources.

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [x] :world_map: Documentation
- [ ] :robot: Test

## Test Plan
I just followed the README to install the foundationDB and tried to insert some records into the database after the pods are running to make sure the foundationDB is working as expected.

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: examples/stable/foundationdb/README.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+building the operator image, we have followed the the upstream `Makefile`
+to create this README that can be used to create a simple foundationDB
+cluster.
+
+
+## Prerequisite 
+
+* Install GO on your machine, see the [Getting Started](https://golang.org/doc/install) guide for more information.
+* Install KubeBuilder and its dependencies on your machine, see [The KubeBuilder Book](https://book.kubebuilder.io/quick-start.html) for more information.
+* Set your $GOPATH.
```
Comment (PrasadG193) [inline examples/stable/foundationdb/README.md]: @viveksinghggits Does this step really required?

File: examples/stable/foundationdb/README.md [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+and we wil be using operator to install foundationDB cluster on our Kubernetes
+cluster.
+
+To get to know about how Kubernetes operators, please follow
+[this link](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).
+
+Steps that are mentioned below can be followed to install the foundationDB operator
+
+* Change your current directory to `$GOPATH/src/github.com` using the command `cd $GOPATH/src/github.com`
+and run `mkdir foundationdb` to create the directory `foundationdb`.
```
Comment (PrasadG193) [inline examples/stable/foundationdb/README.md]: Does it work out of $GOPATH?

Comment (viveksinghggits) [reply examples/stable/foundationdb/README.md]: Hi @PrasadG193 
It doesnt work out of `GOPATH`

File: examples/stable/foundationdb/README.md (hunk 1 above)
Comment (viveksinghggits) [reply examples/stable/foundationdb/README.md]: Hi @PrasadG193 
Yes `kubebuilder` should be installed on the machine.

**Pull request merged**